src/main/java/teammates/common/datatransfer/attributes/SessionAttributes.java

```diff
@@ -19,20 +19,41 @@
      */
     Comparator<SessionAttributes> ASCENDING_ORDER = (session1, session2) -> {
 
+        FeedbackSessionAttributes feedbackSession1 = null;
+        FeedbackSessionAttributes feedbackSession2 = null;
+        Boolean isSession1PrivateFeedback = false;
+        Boolean isSession2PrivateFeedback = false;
+        if (session1 instanceof FeedbackSessionAttributes) {
+            feedbackSession1 = (FeedbackSessionAttributes) session1;
+        }
+        if (session2 instanceof FeedbackSessionAttributes) {
+            feedbackSession2 = (FeedbackSessionAttributes) session2;
+        }
+
         Assumption.assertNotNull(session1.getSessionName());
-        Assumption.assertNotNull(session1.getSessionStartTime());
```

Comment:
Actually, just add a guard clause at the very start. If the session is private, simply compare session name and return. Do the same for `DESCENDING_ORDER`.